examples/apps/cli/main.c

```diff
@@ -133,13 +133,15 @@ int main(int argc, char *argv[])
 #if OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_APP
 void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
 {
-    OT_UNUSED_VARIABLE(aLogLevel);
-    OT_UNUSED_VARIABLE(aLogRegion);
-    OT_UNUSED_VARIABLE(aFormat);
-
     va_list ap;
     va_start(ap, aFormat);
     otCliPlatLogv(aLogLevel, aLogRegion, aFormat, ap);
     va_end(ap);
 }
+
+void otPlatLogLine(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aLogLine)
```

Comment:
I think the best option might be renaming `otPlatLog` to `otPlatLogFormat` or `otPlatLogf` and let the new function be named `otPlatLog`. 
But it seems to be a huge change to the APIs and OT core.

Reply:
I agree it may sound better (more clear) but as you said it would be a big platform API change.

In general, I think/agree that we should aim to keep existing platform and public OT APIs (name and behavior) as they are as much as possible (permit a change only when we have a very good justification for it). Within OT core we can be a lot more flexible.